apps/cowswap-frontend/src/legacy/hooks/useContract.ts

```diff
@@ -9,7 +9,6 @@ import {
   Erc721,
   Erc1155,
   Weth,
-  CoWSwapEthFlowJson,
```

Comment:
Shouldn't it be removed from the abis app? https://github.com/cowprotocol/cowswap/blob/8365242936104e23bba14d970ff6256c0ea55544/libs/abis/src/index.ts#L23

Reply:
Thanks! Deleted